kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/BigQuerySinkTask.java

```diff
@@ -79,6 +69,7 @@
   private TopicPartitionManager topicPartitionManager;
 
   private KCBQThreadPoolExecutor executor;
+  private static final int executor_shutdown_timeout_sec = 30;
```

Comment:
@mtagle , not sure if we wanted to make this a config or not. It's kind of arbitrary right now, reason I didn't add is yet is because it's used for log.warn only, but existing tasks continue to run even after shutdown is signaled so there's no material impact afaik.

Reply:
IMO, 30s is fine as-is here.